src/Control/Middleware/HTTPCacheControlMiddleware.php

```diff
@@ -103,7 +99,7 @@ public function process(HTTPRequest $request, callable $delegate)
      * @config
      * @var string
      */
-    private static $defaultState = self::STATE_DEFAULT;
+    private static $defaultState = self::STATE_ENABLED;
```

Comment:
Sure, I never really liked "default" anyway. I only kept it in to maintain the old behaviour.